src/pages/iou/WaypointEditor.js

```diff
@@ -221,41 +223,45 @@ function WaypointEditor({route: {params: {iouType = '', transactionID = '', wayp
                     danger
                 />
                 <FormProvider
-                    style={[styles.flexGrow1, styles.mh5]}
+                    style={[styles.flexGrow1, styles.mh5, getAddressFormHeight(windowHeight, isOffline && windowWidth < 800)]}
```

Comment:
`isOffline=true` if we are offline and if window width is less than 800
Could you elaborate on why that is?

Reply:
It is for mWeb.
mWeb assumes the width is less than 800px, and we show offline indicator in RHN if it is offline and mWeb.

Reply:
We can use `isSmallScreenWidth` instead. But i still don't think we should calculate AddressFormHeight manually with hardcoded values. What if offline indicator label height changes?

Reply:
I agree wih you to use `isSmallScreenWidth`.
Regarding hard-coded value, I tried to adjust hegith automatically but no luck.

`What if offline indicator label height changes?`
offline indicator label height changes = fontSize + y-padding (16px + 5px * 2).
This is the way what I cacluate the offline indicator label height.

Reply:
I think fontSize for offline indicator is static.

Reply:
> I think fontSize for offline indicator is static

Right now it is, but my point is that's not the correct approach. The list height should be adjusted automatically, we could potentially change offline indicator size/completely remove it/change the save btn size/add another label to the page in the future. 

> I tried to adjust hegith automatically but no luck

I'll look into this myself today or tomorrow, it should be possible in theory, I'll share my findings here

Reply:
Thank you.

Reply:
> I'll look into this myself today or tomorrow, it should be possible in theory, I'll share my findings here
did you find anything to share with me?